lib/__tests__/projets-validator.js

```diff
@@ -65,22 +68,99 @@ const invalidProjet = {
   subventions: [{nom: 'Participation feder', ntr: 'feder'}]
 }
 
+// Test validatePerimetre
+
+test('Valid perimetre, departement', t => {
+  t.is(validatePerimetre('departement:55'), 'departement:55')
+})
+
+test('Invalid perimetre, departement', t => {
+  const error = t.throws(() => {
+    validatePerimetre('departement:666')
+  }, {instanceOf: Error})
+
+  t.is(error.message, 'Territory not found: departement:666')
```

Comment:
Plus simple, ajoute `message: 'Territory not found: departement:666'` dans l'assertion.